### ISSUE TITLE
a11y: accessibility gallery in product page

### ISSUE THREAD

**Issue by ramiy** [woocommerce/woocommerce#18088]
## Description 
The product page has a gallery display under the main image. We can increase the gallery accessibility when using assistive technology. This can be done using a hidden title visible only to screen-readers with `screen-reader-text` CSS class.

## Solution

Code:

`<h3 class="screen-reader-text"><?php esc_html_e( 'Product Gallery', 'woocommerce' ) ?></h3>`

## Location

Between the image and the gallery

![gallery-title](https://user-images.githubusercontent.com/576623/33805503-a41acef2-ddc2-11e7-9ae6-0df9f3185144.png)

**Comment by mikejolley** [comment]
I don't think this one will work or make sense - the gallery output is generated by Flexslider, so any changes in the template/markup won't be shown or visible there. 

Also, I don't see a reason to separate the gallery from the main image in this case. It's a sequence of product images, there should be no distinction between them.

Any changes, if needed, would need adding to flexslider.

Thanks

**Comment by ramiy** [comment]
@mikejolley This request came from an accessibility agency. One of my clients paid for an accessibility report and they told him to add a hidden title above the gallery images.

I did several accessibility projects for my clients, fixing a11y issues - this request make sense. It doesn't mater who creates the gallery output, whether its Flexslider or other Woo template, the gallery should have titles for blind people.

As for separation, we don't separate the gallery from main image. We just add an assistive title to let users know that the following images are part of a product gallery. From an a11y point of view it make sense, even if from a regular point of view it doesn't.

**Comment by mikejolley** [comment]
Right, but the fix itself would need to go into flexslider unless I'm missing something. Perhaps that script itself needs a review too. It takes the images from the HTML and makes the gallery output as it does, so you cannot insert extra content like headings in this case.

**Comment by ramiy** [comment]
@mikejolley Where does WooCommerce generates the gallery? 

Couldn't find the gallery in `templates/single-product/product-image.php`.

**Comment by mikejolley** [comment]
product-thumnails and product-image, and as I said earlier, Flexslider.

**Comment by ramiy** [comment]
Where does the flexslider is initiated?
